SUSYBSMAnalysis/HSCP/test/UsefulScripts/DeDxStudy/DeDxStudy.C

```diff
@@ -192,22 +199,51 @@ void DeDxStudy(string DIRNAME="COMPILE", string INPUT="dEdx.root", string OUTPUT
       InitBaseDirectory();
       GetSampleDefinition(samples , DIRNAME+"/../../AnalysisCode/Analysis_Samples.txt");
       stSample& sample = samples[JobIdToIndex(SampleId, samples)];
-      isData = (sample.Type==0);
+      isData   = (sample.Type==0);
+      isSignal = (sample.Type==2);
       GetInputFiles(sample, BaseDirectory, FileName, 0);
    }
 
-   TH3F* dEdxTemplates    = NULL;
-   TH3F* dEdxTemplatesInc = NULL;
+//   TH3F* dEdxTemplates    = NULL;
+//   TH3F* dEdxTemplatesInc = NULL;
+   TH3F* dEdxTemplatesPO      = NULL;
```

Comment:
Way to many templates which all contains the same information.
you must have only 3
SP
SP_Split
SP_Split_In
Realize the amount of memory consumed by these histograms?